tests/tensorbox/test_tensorbox.py

```diff
@@ -0,0 +1,265 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for TensorBox class"""

```

Comment:
do i understand correctly, the tests of the base class are just done using numpy arrays?

Reply:
hmm, wouldn't it be cleaner to test the numpy tensor box module instead? and just add a few minimal tests for the tensorbox base class?

Reply:
I couldn't work out how, since it is an abstract base class and thus doesn't have all the methods implemented 🤔